modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
I accidentally left EventTracker in this commit (it should be deleted). I spoke with @mandragorn  and @catbieber and we've concluded that it would be best to trigger all the timing events to Torso.Events so that any customized event tracker can listen to Torso.Events to capture all render/router/fetch timings. Do you have thoughts about this?

Would you still prefer to trigger each event on 'this', and add an 'on' to the constructor and register the events to Events?